validator_config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
this file should not be comitted!  Likely need to add it to .gitignore or similar.